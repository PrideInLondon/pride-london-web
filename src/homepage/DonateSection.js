import React from 'react'
import { Row } from '../components/grid'
import donateSectionLeft from './donateSectionLeft.svg'
import donateSectionRight from './donateSectionRight.svg'
import donateSectionRightMobile from './donateSectionRightMobile.svg'
import DonateButtonsGroup from './DonateButtonsGroup'
import {
  DonateSectionsWrapper,
  HeadDonate,
  BackgroundLeft,
  BackgroundRight,
  LayerContent,
} from './DonateSection.styles'

const DonateSection = () => (
  <DonateSectionsWrapper>
    <BackgroundLeft>
      <img src={donateSectionLeft} alt="bg donate" />
    </BackgroundLeft>
    <BackgroundRight>
      <picture>
        <source media="(min-width: 768px)" srcSet={donateSectionRight} />
        <img src={donateSectionRightMobile} alt="Mobile background" />
      </picture>
    </BackgroundRight>
    <LayerContent>
      <Row>
        <HeadDonate>Make a single donation</HeadDonate>
      </Row>
      <Row>
        <DonateButtonsGroup />
      </Row>
    </LayerContent>
  </DonateSectionsWrapper>
)

export default DonateSection
