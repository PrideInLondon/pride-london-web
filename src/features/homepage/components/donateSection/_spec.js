import React from 'react'
import { shallow } from 'enzyme'
import DonateSection from '.'

describe(DonateSection.name, () => {
  it('volunteer snapshot', () => {
    const wrapper = shallow(<DonateSection />)
    expect(wrapper).toMatchSnapshot()
  })
})
