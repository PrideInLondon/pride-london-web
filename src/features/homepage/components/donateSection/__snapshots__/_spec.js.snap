// Jest Snapshot v1, https://goo.gl/fbAQLP

exports[`DonateSection volunteer snapshot 1`] = `
<styles__DonateSectionsWrapper>
  <styles__BackgroundLeft>
    <img
      alt="bg donate"
      src=""
    />
  </styles__BackgroundLeft>
  <styles__BackgroundRight>
    <picture>
      <source
        media="(min-width: 768px)"
        srcSet=""
      />
      <img
        alt="Mobile background"
        src=""
      />
    </picture>
  </styles__BackgroundRight>
  <styles__LayerContent>
    <Row
      flexWrap={null}
      ml={null}
      mr={null}
      mx={null}
    >
      <styles__HeadDonate>
        Make a single donation
      </styles__HeadDonate>
    </Row>
    <Row
      flexWrap={null}
      ml={null}
      mr={null}
      mx={null}
    >
      <DonateButtonsGroup />
    </Row>
  </styles__LayerContent>
</styles__DonateSectionsWrapper>
`;
